druntime/Makefile

```diff
@@ -497,10 +497,15 @@ $(addsuffix /.run,$(filter-out test/shared,$(ADDITIONAL_TESTS))): $(DRUNTIME)
 test/shared/.run: $(DRUNTIMESO)
 endif
 
+ifeq (linux,$(OS))
+    LDD = ldd
+    IS_MUSL != $(LDD) --version 2>&1 | grep -q musl && echo 1
```

Comment:
That's IMO some weird/unintuitive syntax; found it in https://www.gnu.org/software/make/manual/html_node/Setting.html. I think the `$(shell …)` wrapper is easier to understand.

Reply:
The reason for using `!=` is that it's part of posix, not a gnu extension. It's also documented on https://www.gnu.org/software/make/manual/html_node/Shell-Function.html.

I don't object to changing it, if the gnu syntax is preferred, since the makefiles already rely on it